sources/packages/backend/apps/api/src/route-controllers/dynamic-form/dynamic-form.controller.ts

```diff
@@ -23,12 +28,46 @@ export class DynamicFormController extends BaseController {
     super();
   }
 
+  /**
+   * List all form definitions that contains the tag 'common' ordered by title.
+   * @returns list of form definitions.
+   */
+  @Roles(Role.AESTFormEditor)
```

Comment:
This endpoint must be allowed only for AEST azp.

Reply:
It will be since we are moving to AEST-specific controller as mentioned here https://github.com/bcgov/SIMS/pull/4674#discussion_r2082489483